packages/docs-reanimated/docs/layout-animations/keyframe-animations.mdx

```diff
@@ -96,117 +25,68 @@ const keyframe = new Keyframe({
+}
 ```
 
-## How to use keyframe animations?
+### Arguments
 
-Currently, you can define animations using keyframes only for entry and exit animations.
+#### `definitions`
 
-### 1. Choose Animated Component which entering or exiting you want to animate
+An object, passed as a first argument that contains definitions of your animation.
+The object keys should be within range `0-100` and correspond to animation progress.
+The object values should consist of style props and/or [easing](/docs/animations/withTiming/#easing) function. If easing property is not provided, it defaults to `Easing.linear`.
 
-```js
-// AnimatedComponent - component created by createAnimatedComponent or imported from Reanimated
-// keyframe - Keyframe object
-<AnimatedComponent exiting={keyframe} />
-```
+- `0` or `from`: Initial state of the object.
+- Middle points (e.g., `45`): Intermediate states of the object.
+- `100` or `to`: Final state of the object.
 
-### 2. Customize the animation
+To `0` assign the style, you want for your object at the beginning of the animation and to `100` assign the style you want for your object to have at the end of the animation.
 
-```js
-<AnimatedComponent exiting={keyframe.duration(3000).delay(200)} />
-```
+## Remarks
 
-## Available modifiers
+- Providing keyframe `0` or `from` is required as it contains the initial state of the object you want to animate.
+- Ensure you provide the initial value for all style properties you want to animate in other keyframes.
+- Do not provide both `0` and `from`, or `100` and `to` keyframes, as it will result in a parsing conflict.
+- If you want to animate transform style, make sure that all properties in the transformation array are in the same order in all keyframes.
 
-The order of modifiers doesn't matter.
+## Modifiers
 
-### duration
-
-default: 500
-How long the animation should last.
+```javascript
+keyframe
+  .duration(1000)
+  .delay(500)
+  .reduceMotion(ReduceMotion.Never)
+  .withCallback((finished) => {
+    console.log(`finished without interruptions: ${finished}`);
+  });
+```
 
-### delay
+- `.duration(durationMs: number)` is the length of the animation (in milliseconds). Defaults to `500`.
+- `.delay(durationMs: number)` is the delay before the animation starts (in milliseconds). Defaults to `0`.
+- `.reduceMotion(reduceMotion: ReduceMotion)` determines how the animation responds to the device's reduced motion accessibility setting.
+- `.withCallback(callback: (finished: boolean) => void)` is the callback that will fire after the animation ends. Sets `finished` to `true` when animation ends without interruptions, and `false` otherwise.
 
-default: 0
-Allows to start with a specified delay.
+## Example
 
-### reduceMotion
+<ExampleVideo
+  sources={{
+    android:
+      '/react-native-reanimated/recordings/layout-animations/keyframe_android.mov',
+    ios: '/react-native-reanimated/recordings/layout-animations/keyframe_ios.mov',
+  }}
+/>
```

Comment:
I'd move the `Modifiers` and `Examples` sections ***before*** the `Remarks`. Remarks are like some additional comments and pitfalls to be aware of, but they assume that the user knows the API, thus I think it is better to have them after Modifiers and Examples.

Reply:
I left it there because they were talking about the usage of keys, but it's a good idea to move them at the end